config/initializers/kaminari_config.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+Kaminari.configure do |config|
+  config.default_per_page = 5
+  # config.max_per_page = nil
+  # config.window = 4
+  # config.outer_window = 0
+  # config.left = 0
+  # config.right = 0
+  config.page_method_name = :page
+  config.param_name = :page
```

Comment:
What's this field used for? maybe we don't need to specify the page param name here and in the controller

Reply:
config.page_method_name = :page  => in order to play nice with existing page method or association or scope or any other plugin that defines page method on your models.